### DISCUSSION TITLE
Add new filter to oscp?

### DISCUSSION THREAD

**Discussion by aaronjstanley** [bullhorn/career-portal#410] [Q&A]
I have a client wanting to change the state filter to country.  Is there a way to add/edit/remove filters?  I'm not having much luck finding any documentation around this.  Thanks!

**Comment by charlesabarnes** [comment]
There is not a simple configuration for this change.  This requires code modification to the [sidebar](https://github.com/bullhorn/career-portal/tree/master/src/app/sidebar).  Please let me know if you have any additional questions.

**Reply by nate-gel** [reply]
Hi @charlesabarnes do you have any guides or docs on what modifications are required in order to make this change?

**Reply by SherpaGreg** [reply]
I am trying to adjust the filters as well.  Has anyone figured out how to do this?

**Reply by pbearne** [reply]
Hi All 
The matador Jobs has this as setting change the location source 
If you are not got access to a react dev this may be a better solution

**Comment by hutchred** [comment]
Anyone do this yet? I am currently looking into making this change on our portal to accommodate two countries.

**Reply by pbearne** [reply]
Reach out to Matador Jobs if this is a must-have our solution can do this
